.ci_support/osx_64_.yaml

```diff
@@ -7,7 +7,7 @@ channel_targets:
 fortran_compiler:
 - gfortran
 fortran_compiler_version:
-- '9'
+- '11'
```

Comment:
I would be curious to see if the mac binaries now work with the newer gfortran version